hazelcast/src/main/java/com/hazelcast/monitor/impl/HotRestartStateImpl.java

```diff
@@ -56,12 +65,16 @@ public void fromJson(JsonObject json) {
         final String jsonBackupTaskState = getString(json, "backupTaskState", null);
         final int jsonBackupTaskCompleted = JsonUtil.getInt(json, "backupTaskCompleted", 0);
         final int jsonBackupTaskTotal = JsonUtil.getInt(json, "backupTaskTotal", 0);
+        final boolean jsonIsHotBackupEnabled = JsonUtil.getBoolean(json, "isHotBackupEnabled", false);
         backupTaskStatus = jsonBackupTaskState != null ? new BackupTaskStatus(BackupTaskState.valueOf(jsonBackupTaskState),
                 jsonBackupTaskCompleted, jsonBackupTaskTotal) : null;
+        isHotBackupEnabled = jsonIsHotBackupEnabled;
```

Comment:
Why do we need `jsonIsHotBackupEnabled`, looks like it's not used anywhere? I think, we can get rid of `jsonIsHotBackupEnabled` and assign the parsed boolean directly to `isHotBackupEnabled`.